server/queries/updateEmployeesWithSlackID.js

```diff
@@ -0,0 +1,32 @@
+const axios = require('axios');
+const addSlackInfo = require('./addSlackInfo');
+
+const updateEmployeesWithSlackID = async (db) => {
+  const slackOAuthToken = process.env.SLACK_OAUTH;
```

Comment:
Please use the `getSLACK_OAUTH` functionality you made! This helps to ensure a project is setup correctly before trying to use things.